PipelinesTasks/PRMetrics/buildAndReleaseTask/wrappers/ProcessWrapper.ts

```diff
@@ -0,0 +1,7 @@
+class ProcessWrapper {
+    public log(input: string) {
```

Comment:
I'd suggest calling this `write` instead to keep it consistent with the wrapped method.